README.md

```diff
@@ -1,5 +1,114 @@
-# dscrait.tech
+
+<h1 align="center"> dscrait.tech README</h1>
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/dscrait/dscrait.tech">
+    <img src="https://raw.githubusercontent.com/mrunalvilas/dscrait.tech/update_readme_docs/logo/DSCLOGO.PNG" width="300" height="300">
+  </a>
+</p>
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+Index | Content Heading
+------------|--------------------------------
+ 1 | [About the Project](#about-the-project)
+ 2 | [Built With](#built-with)
+ 3 | [Prerequisites](#prerequisites) 
+ 4 | [Installation](#Installation)
+ 5 | [License](#license)
+ 6 | [Contact](#contact)
+ 7 | [Acknowledgements](#acknowledgements)
+ 
+<hr />
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+
 
 The website for DSC-RAIT - [dscrait.tech](https://dscrait.tech)
 
 DSC-RAIT is a community of budding developers, entrepreneurs, and innovators, getting ready to solve real-world problems. Our mission is to inculcate an innovative mindset in our members to help our local community as well as contributing towards solving global crises. We hope to build impactful software and guide the future leaders of our world.
+
+
+### Built With
+Major frameworks that are used to  built this project are listed below:
+* [Typescript](https://www.typescriptlang.org/)
+* [NextJS](https://nextjs.org/)
+* [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+* [ReactJS](https://reactjs.org/)
+* [NodeJS](https://nodejs.org/en/)
+* [Bootstrap](https://getbootstrap.com)
+* [fortawesome](https://fortawesome.com/)
+
+
+
+
+### Prerequisites
+
+Following  Important softawares are needed for this project
+
+* nodejs
+```sh
+you can download nodejs based on your operating system
+```
+from here [Download NodeJS](https://nodejs.org/en/download/)
+
+* npm
+```sh
+npm install npm@latest -g
+```
+
+
+
+### Installation
+
+1. Clone the repo
+```sh
+git clone https://github.com/dscrait/dscrait.tech.git
+```
+2. Install NPM packages
+```sh
+npm install
+```
+3. Run the website in your localhost by typing this command
+```sh
+npm run dev
+```
+
+<!-- LICENSE -->
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+<!-- CONTACT -->
+## Contact
+
+DSC-RAIT: [dsc.rait@gmail.com](dsc.rait@gmail.com) 
+
+Project Link: [https://github.com/dscrait/dscrait.tech](https://github.com/dscrait/dscrait.tech)
+
+You can join our Discord server for more details:  [Discord](https://discord.com/channels/743724391296270377/749248351563677697)
```

Comment:
I don't think this link is an invite link, please change it to the invite link sent to your email when you enrolled